modules/core/src/main/scala/org/scalasteward/core/buildtool/sbt/SbtAlg.scala

```diff
@@ -110,7 +110,7 @@ final class SbtAlg[F[_]](config: Config)(implicit
       projectDir = buildRootDir / "project"
       files0 <- (
         fileAlg.walk(buildRootDir, 1).filter(_.extension.contains(".sbt")) ++
-          fileAlg.walk(projectDir, 1).filter(_.extension.exists(Set(".sbt", ".scala")))
+          fileAlg.walk(projectDir).filter(_.extension.exists(Set(".sbt", ".scala")))
```

Comment:
I think depth `2` is enough to scan `project/project`.

```suggestion
          fileAlg.walk(projectDir, 2).filter(_.extension.exists(Set(".sbt", ".scala")))
```

Reply:
Believe it or not, I do have a (non public) project which has a `project/project/project/plugins.sbt` file. So I personally would prefer a depth of 3 😉 
```suggestion
          fileAlg.walk(projectDir, 3).filter(_.extension.exists(Set(".sbt", ".scala")))
```

Reply:
Oh... then 3 is good to go.